src/org/zaproxy/zap/extension/pscanrulesAlpha/XPoweredByHeaderInfoLeakScanner.java

```diff
@@ -71,7 +71,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		    				"", // Other info
 		    				getSolution(), //Solution
 		    				getReference(), //References
-		    				xpbDirective,	// Evidence - Return the X-Powered-By Header info
+		    				"X-Powered-By: " + xpbDirective, // Evidence - Return the X-Powered-By Header info

```

Comment:
Although unlikely(?), as is now the highlight will not work if the header has a different case than the evidence. (Not a big problem IMHO, since the alert would be raised anyway.)